src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -17,10 +17,17 @@
 
 module("dialogs-opening_or_not", {setup: prepareUnitTestModule});
 
+var DIALOG_EXPECT_OPENED = "Expect this dialog to have been opened. " +
+    "Expect it with all of your heart.";
+var DIALOG_EXPECT_NOT_OPENED = "Nope. The dialog should not have opened.";
```

Comment:
@winhamwr these are now not properties of `dialogTestHelper` because it was renamed to be much longer. They are globals now.